tests/service/FscTests.fs

```diff
@@ -163,10 +163,10 @@ let compile isDll debugMode (assemblyName : string) (ext: string) (code : string
         |]
 
     ensureDefaultFSharpCoreAvailable tmp
-        
```

Comment:
as a note, is easier to review if whitespace are not changed. can that be doen in another pr 😄